CONTRIBUTING.md

```diff
@@ -30,6 +32,52 @@ Once you have chromedriver installed and available in `PATH` you can re-generate
 
 To add a new test case add another HTML file to `/test_fixtures` following the current tests as a template for new tests.
 
+### Writing tests
+
+1. All tests should be wrapped in a module called `tests`, to ensure they are not compiled unless testing
+
+```rs
+#[cfg(test)]
+mod tests {
+    // Place tests here
+}
+```

```

Comment:
I'd recommend to indent these with 4 spaces, otherwise some Mardown renderers will terminate the enumeration here and the `2.` will show up as `1.` again.

The same applies for the other items.